dracut.conf.d/50-ima.conf

```diff
@@ -0,0 +1,2 @@
+# turn on ima modules
```

Comment:
What does `ima` stand for?

Reply:
Integrity Measurement Architecture

I also updated the comment in the config file.